uk.co.compucorp.civicrm.hremails/email-templates/src/pages/leave-request.html

```diff
@@ -0,0 +1,69 @@
+---
```

Comment:
this is used by the foundation framework to generate the final email (`dist/leave-request.html`), pls check the documentation